.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: 2.6
+  Exclude:
+    - bin/bundle
+Style/Documentation:
+  Enabled: false
+Metrics/ClassLength:
+  Enabled: true
+  Max: 150
+Metrics/AbcSize:
+  Enabled: true
+  Max: 20
+Metrics/LineLength:
+  Enabled: true
+  Max: 160
```

Comment:
Fair point. TBH I just copied it from the Homeday repo. I adapted the settings now to something more sensible. Since I fixed all rubocop warnings I went for a line length of 120. This way we have 0 warnings without me spending too much time adding line-breaks :)